crates/sp-domains-fraud-proof/src/verification.rs

```diff
@@ -372,22 +370,22 @@ where
     })?;
 
     let derived_domain_block_hash = sp_domains::derive_domain_block_hash::<DomainHeader>(
-        bad_receipt.domain_block_number,
-        bad_receipt.domain_block_extrinsic_root,
+        *bad_receipt.domain_block_number(),
+        *bad_receipt.domain_block_extrinsics_root(),
         state_root,
         parent_domain_block_hash,
         digest,
     );
 
-    if bad_receipt.domain_block_hash == derived_domain_block_hash {
+    if *bad_receipt.domain_block_hash() == derived_domain_block_hash {
         return Err(VerificationError::InvalidProof);
     }
 
     Ok(())
 }
 
-/// Verifies invalid block fees fraud proof.
-pub fn verify_invalid_block_fees_fraud_proof<
+/// Verifies invalid transfers fraud proof.
+pub fn verify_invalid_transfers_fraud_proof<
```

Comment:
Is there a reason the order of these functions was swapped? It makes them harder to review.